doc(setting): eventual danger zone setting update (#858)
longhorn/website: content/docs/1.6.0/advanced-resources/deploy/taint-toleration.md

   * If you install Longhorn using Helm, you can change the Helm values for `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file.
    Then install the chart

> 2. Set taint tolerations for system managed components (Instance manager, Engine image, CSI driver, etc.)

Reviewer: ```suggestion
2. Set taint tolerations for system-managed components (for example, Instance Manager, CSI Driver, and engine images).
```

> **Warning**:
>

> > Before modifying the toleration settings, suggest detaching all Longhorn volumes.

Reviewer: ```suggestion
> Detach all Longhorn volumes before modifying the toleration settings.
```

Author: Modified. Thanks

> > To apply a setting immediately, stop all workloads and detach all Longhorn volumes before modifying the toleration settings.

Reviewer: ```suggestion
> To ensure that your preferred toleration settings are immediately applied, stop all workloads and detach all Longhorn volumes before configuring the settings.
```

Author: Updated. Thanks

>
> Since all Longhorn components will be restarted, the Longhorn system is unavailable temporarily.
>

> > For Longhorn volumes all detached, the customized setting will be applied to system-managed components immediately.
> > For one or more volumes in use, the customized setting will be applied to the Instance Manager without engine and replica instances running, and you are required to reconfigure this setting when all volumes are finally detached.

Reviewer: ```suggestion
> When all Longhorn volumes are detached, the customized settings are immediately applied to the system-managed components.
> When one or more Longhorn volumes are still attached, the customized settings are applied to the Instance Manager only when no engines and replica instances are running. You might be required to reconfigure the settings after detaching the remaining volumes.
```

Author: Modified. Thanks

>
> Don't operate the Longhorn system while toleration settings are updated and Longhorn components are being restarted.

1. Prepare

>    Suggest stopping all workloads and detaching all Longhorn volumes.

Reviewer: ```suggestion
   For best results, stop all workloads and detach all Longhorn volumes.
```

>    To apply a setting immediately, stop all workloads and detach all Longhorn volumes before applying it.

Reviewer: ```suggestion
   To ensure that your preferred settings are immediately applied, stop all workloads and detach all Longhorn volumes before configuring the settings.
```

Author: Updated. Thanks

> Before modifying the toleration settings, users should make sure all Longhorn volumes are `detached`.
>
> Since all Longhorn components will be restarted, the Longhorn system is unavailable temporarily.

> > If there are running Longhorn volumes in the system, this means only the instance manager component will be restarted eventually and you might need to set this setting again when all Longhorn volumes are `detached`.

Reviewer: ```suggestion
> When volumes are in use, only Instance Manager is restarted so you might be required to configure this setting again when all volumes are finally detached.
```

Author: Modified. Thanks.
